make/test.mk

```diff
@@ -181,6 +184,23 @@ else
 	$(MAKE) print-local-debug-info  HOST_NS=${HOST_NS} MEMBER_NS=${MEMBER_NS} MEMBER_NS_2=${MEMBER_NS_2} REGISTRATION_SERVICE_NS=${REGISTRATION_SERVICE_NS}
 endif
 
+.PHONY: check-go-junit-report
+
+check-go-junit-report:
+	@command -v ./go-junit-report >/dev/null 2>&1 || { echo "go-junit-report is not installed. Installing..."; GOBIN=$(PWD) go install github.com/jstemmer/go-junit-report/v2@latest; }
+	@echo "go-junit-report version:" && ./go-junit-report -version
```

Comment:
I would rather avoid installing the binary in the folder of the repository - let's install it either in /tmp folder or somewhere else.